operations/docker/php72_integration/install_magento_data.sh

```diff
@@ -22,6 +22,8 @@ set -x
 
 sudo service mysql start -- --initialize-insecure --skip-grant-tables --skip-networking --protocol=socket
 composer install
+sudo mysql -u root -e "USE mysql;UPDATE user SET plugin='mysql_native_password' WHERE User='root';FLUSH PRIVILEGES;"
+sudo service mysql restart
```

Comment:
when you restart do you need to specify insecure / skip grant and skip netowrking?

Reply:
I am facing no errors omitting this